dist/img/diagonal.svg

```diff
@@ -0,0 +1 @@
+<svg id="diagonal-divider" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1440.28 103.99"><defs><style>.a{fill:none;stroke-miterlimit:10}</style></defs><title>diagonal</title><path  class="a" d="M.14 2l1440 100"/></svg>
```

Comment:
this uses an `id` but is on pages more than once. should be a class (which should survive our svgo config fine I think)